src/acl/FilledChecklist.h

```diff
@@ -87,11 +87,12 @@ class ACLFilledChecklist: public ACLChecklist
     char *snmp_community;
 #endif
 
+    // TODO: RefCount errors; do not ignore them because their "owner" is gone!
     /// TLS server [certificate validation] errors, in undefined order.
     /// The errors are accumulated as Squid goes through validation steps
     /// and server certificates. They are cleared on connection retries.
     /// For sslproxy_cert_error checks, contains just the current/last error.
-    const Security::CertErrors *sslErrors;
+    CbcPointer<Security::CertErrors> sslErrors;
```

Comment:
The loss of `const` here and elsewhere is unfortunate but necessary because CbcPointer calls non-const toCbdata().